modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md

```diff
@@ -0,0 +1,36 @@
+# Cohesive Zone Model
```

Comment:
Change to `# Cohesive Zone Master Action System` to be consistent with other documentation.

Reply:
ok